docs/source/index.rst

```diff
@@ -286,6 +286,7 @@ This project is licensed under an all rights reserved licence.
     implementations/labeling_trend_scanning
```

Comment:
When we have finished all the implementations, we will need to come back to this to re order it.